testing:
  model_abspath: /data/catkin_ws/src/drl_navigation/models/revit_no_object/SAC_240000.zip
  num_episodes: 100